### PR TITLE
Add a rnn example on mnist dataset using tf library

### PR DESCRIPTION
**Pull request by imsheridan**
A Recurrent Neural Network (LSTM) implementation example using TensorFlow library.
This example is using the MNIST database of handwritten digits (http://yann.lecun.com/exdb/mnist/)
Links:
    [Long Short Term Memory](http://deeplearning.cs.cmu.edu/pdfs/Hochreiter97_lstm.pdf)
    [MNIST Dataset](http://yann.lecun.com/exdb/mnist/).

Training and evaluation log:
Extracting /tmp/mnist_data/train-images-idx3-ubyte.gz
Extracting /tmp/mnist_data/train-labels-idx1-ubyte.gz
Extracting /tmp/mnist_data/t10k-images-idx3-ubyte.gz
Extracting /tmp/mnist_data/t10k-labels-idx1-ubyte.gz
Step 1, Minibatch Loss= 2.5586, Training Accuracy= 0.258
Step 200, Minibatch Loss= 0.2419, Training Accuracy= 0.930
Step 400, Minibatch Loss= 0.1863, Training Accuracy= 0.938
Step 600, Minibatch Loss= 0.1000, Training Accuracy= 0.969
Step 800, Minibatch Loss= 0.0935, Training Accuracy= 0.977
Step 1000, Minibatch Loss= 0.0773, Training Accuracy= 0.969
Step 1200, Minibatch Loss= 0.0500, Training Accuracy= 0.984
Step 1400, Minibatch Loss= 0.0550, Training Accuracy= 0.977
Step 1600, Minibatch Loss= 0.0615, Training Accuracy= 0.984
Step 1800, Minibatch Loss= 0.0635, Training Accuracy= 0.977
Step 2000, Minibatch Loss= 0.0550, Training Accuracy= 0.992
Training Finished!
Testing Accuracy: 0.992188

### CODE REVIEW

**Comment by drpngx** [comment]
Closing as this is moved to another repo. Thank you!

**Comment by imsheridan** [comment]
@drpngx It seems mis-closed this one, I assume you mean to close this one [16272](https://github.com/tensorflow/tensorflow/pull/16272), right?

**Comment by drpngx** [comment]
Woops sorry. Reopened.

**Comment by imsheridan** [comment]
@MarkDaoust Could you please help kindly have a look?

**Comment by imsheridan** [comment]
Close this PR since seems no admin review this.
